packages/main-ui/src/pages/surveys/new/controller.rs

```diff
@@ -20,14 +28,67 @@ pub struct Controller {
 
     questions: Signal<Vec<Question>>,
     total_survey_types: Signal<Vec<String>>,
+    current_step: Signal<CurrentStep>,
+
+    panels: Signal<Vec<PanelV2Summary>>,
+    panel_resource: Resource<QueryResponse<PanelV2Summary>>,
+    client: Signal<PanelV2Client>,
+    org_id: Signal<String>,
+    selected_panels: Signal<Vec<SelectedPanel>>,
+    maximum_panel_count: Signal<Vec<u64>>,
+    total_panel_members: Signal<u64>,
+    popup_service: PopupService,
+
+    translates: Signal<SurveyNewTranslate>,
+}
+
+#[derive(Debug, Clone, PartialEq, Copy)]
+pub enum CurrentStep {
+    CreateSurvey,
+    SettingPanel,
+}
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct SelectedPanel {
+    pub id: String,
+    pub name: String,
+    pub total_count: u64,
 }
 
 impl Controller {
     pub fn new(lang: dioxus_translate::Language) -> Self {
-        let translates: SurveyNewTranslate = translate(&lang.clone());
+        let translates: SurveyNewTranslate = translate(&lang);
+        let login_service: LoginService = use_context();
+        let org_id = match login_service.get_selected_org() {
+            Some(v) => v.id,
+            None => "".to_string(),
+        };
+        let client = PanelV2::get_client(&crate::config::get().api_url);
+
+        let org_id_copy = org_id.clone();
+
+        let panel_resource = use_resource({
+            let client = client.clone();
+            move || {
+                let client = client.clone();
+                let org_id = org_id.clone();
+
+                async move {
+                    //FIMXE: fix to get total data
+                    let query = PanelV2Query::new(100).with_page(1);
```

Comment:
```suggestion
            let client = client.clone();
            let page = page();
            let size = 100;
            move || {
                let client = client.clone();
                let org_id = org_id.clone();

                async move {
                    //FIMXE: fix to get total data
                    let query = PanelV2Query::new(size).with_page(page);
```
이렇게 하면 나중에 page 만 바꾸면 자동으로 survey 도 바뀝니다.
pannel controller 구현 참고해주세요.

Reply:
넵 수정했습니다!